src/rating-summary/ratingSummary.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC, ReactElement, useMemo } from 'react';
+import { ISummaryProp } from './types';
+import classes from './styles.module.scss';
+import starImg from '../assets/star-grey.svg';
+import { Elements, RatingIds } from '../constants';
+
+const RatingSummary: FC<ISummaryProp> = (props): JSX.Element => {
+  const { ratings,
+    renderLabel,
+    showCount = true,
+    showAnimation = true,
+    styles={}
+  } =props;
+
+  const getRatingLabelFromRatingId = (ratingId: string): ReactElement => {
+    if (renderLabel) return (renderLabel(ratingId));
+    let defaultLabel = '';
+    switch (ratingId) {
+    case RatingIds.one:
+      defaultLabel = '1'
```

Comment:
Move the default Label to a constant file

Reply:
defaultLabel is a scoped variable actually, i think the name is confusing changing the variable name